docs/csa/audit-and-compliance.md

```diff
@@ -0,0 +1,170 @@
+
+#### Perform a PCI compliance audit
+
+This example shows how to use the PCI app for Palo Alto Networks to perform a PCI audit for requirement 4: "Encrypt transmission of cardholder data across open, public networks".
+
+Before you can perform the audit, collect logs and install the app for the version of Palo Alto Networks you use:
+* [PCI Compliance for Palo Alto Networks 9](/docs/integrations/pci-compliance/palo-alto-networks-9/)
+* [PCI Compliance for Palo Alto Networks 10](/docs/integrations/pci-compliance/palo-alto-networks-10/)
+
+To perform the audit:
+
+1. In the left navigation pane, click your **Personal** folder, then expand the PCI Compliance app folder.
+1. Open the **PCI Compliance For Palo Alto Networks - PCI Req 02, 04 - Insecure Data In Transit** dashboard.
+1. Find the **Insecure Allowed Traffic by Target Port and Involved Host** panel. 
+1. In the top right corner, click the three-dot button and select **Open in Log Search**.
+1. Next to the clock icon, select **Last 24 Hours**. (You may have to select a longer time frame to get results.)
+1. Click the magnifying glass icon or press Enter to start the search.
+1. Click the **Messages** tab in the results. Here you’ll see the details of every log message that match the query. You’ll see columns for the timestamp, port, IP address, number of bytes sent, number of bytes received, and many other fields. 
+1. Click the **Aggregates** tab.
+1. Click the Table icon. The aggregates table will only show the potential incidents. 
+
+For example, there may be log messages on destination `port 80`. While these might not be true incidents, the query has flagged them as worthy of investigation. You can try to determine which network this IP address is on, and whether or not the data is insecure. Recall requirement 4: "Encrypt transmission of cardholder data across open, public networks." If this data is on an open, public network, you'll need to make sure it’s encrypted so you can meet this compliance standard.
+
+## Build queries for audit and compliance
+
+You can use Cloud Security Analytics to build queries to assist you in audit and compliance tasks. 
+
+### Use case: Audit AWS root for compliance
+
+In this use case, we'll show you how to query AWS CloudTrail data for any messages that contain `root`, `su`, or `sudo` keywords indicating root account access.
+
+#### What is a root account?
+
+Most operating systems have a small handful of super user accounts. These accounts may be known as su (super user), root, or admin. These accounts have special access beyond what normal accounts have. They can often add and delete data, create and delete users, and perform other privileged actions. 
```

Comment:
this might be misunderstood by someone w/EN as their second language

```suggestion
Most operating systems have a small number of super user accounts. These accounts may be known as su (super user), root, or admin. These accounts have special access beyond what normal accounts have. They can often add and delete data, create and delete users, and perform other privileged actions. 
```